<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font: 11pt Poppins;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 40px;
border-radius: 20px;
border: 1px solid #00007f;
background-color: white;
color: #00007f;
font-weight: 500;
font-size: 11px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widgetKeyboard" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>185</y>
      <width>480</width>
      <height>135</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>6</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Lugar:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputPlace">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>2</y>
       <width>180</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 12pt &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Fuente:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;</string>
     </property>
     <item>
      <property name="text">
       <string>Escoja una opción</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Acuífero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arroyo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Depósito de agua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Embalse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Estanque</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fuente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Llave</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lluvia recolectada</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manantial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Planta de tto de agua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pozo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Río</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tanque de almacenamiento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Torre de agua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Otro</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 11pt;
background-color: transparent;
height: 21px;
</string>
     </property>
     <property name="text">
      <string>¿Llovió recientemente?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gpsBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: #00007f;
color: white;
border: none;
border-radius: 15px;
font-size: 11pt;
</string>
     </property>
     <property name="text">
      <string>Localizar con GPS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Lattitud, longitud&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>backBtn</zorder>
   <zorder>widgetKeyboard</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
